x-pack/legacy/plugins/ml/public/management/jobs_list/components/jobs_list_page/_expanded_row.scss

```diff
@@ -1,3 +1,8 @@
+.disabledLink {
```

Comment:
Should probably be prefixed with `ml`.

Reply:
Removed the class as it's no longer needed since now just the plain id is returned instead of a disabled link. e8d66134bbd424cd3231493db8bb0f613af47525